### DISCUSSION TITLE
Add a way to roll back updates

### DISCUSSION THREAD

**Discussion by bigbanana80** [zen-browser/desktop#3253] [Ideas]
1.0.1-a.21 (Firefox 133.0) (64-bit)

This is the third time that this browser updated and it either became unusable or annoying to use, this time it broke my sideberry zen mod which is essential for me, so I can no longer use this browser in a comfy way. a way to roll back updates would be so good since this browser has a tendency to break whenever it gets updated.

For now, i enable compact mode with sidebar hidden as a workaround.

**Comment by kuqquk** [comment]
You can find all previous version from the release: [https://github.com/zen-browser/desktop/releases](https://github.com/zen-browser/desktop/releases)
